tests/integration/reviewIntegrationTests.js

```diff
@@ -0,0 +1,321 @@
+'use strict';
+
+var chai = require('chai'),
+    chaiAsPromised = require('chai-as-promised'),
+    request = require('supertest'),
+    app = require('../../server'),
+    User = require('../../app/schema/userModel'),
+    Artist = require('../../app/schema/artistModel'),
+    Album = require('../../app/schema/albumModel'),
+    Review = require('../../app/schema/reviewModel');
+
+chai.use(chaiAsPromised);
+var expect = chai.expect;
+
+describe('The Reviews controller', function() {
+  before(function(done) {
+    var admin = new User({
+      username: 'admin',
+      password: 'adminSecret',
+      email: 'admin@example.com',
+      auth_level: 'Admin'
+    });
+
+    var kanye = new Artist({
+      name: 'Kanye West',
+      alpha_name: 'westkanye'
+    });
+
+    var jayz = new Artist({
```

Comment:
I don't think this `jayz` is relevant at all (lol), so you can delete this and other `jayz` references